### DISCUSSION TITLE
Change default checksum header on artifactories; or if possible send extra checksum header?

### DISCUSSION THREAD

**Discussion by anitabee** [goreleaser/goreleaser#2152] [Q&A] [answered]
Hi all,

I'm trying to figure out if it's possible to send multiple Checksum headers for artifactories, but even when I'm just trying to change the default with something like below, call to Artifactory again uses `X-Checksum-Sha256` header:

```
checksum:
  algorithm: Sha1

artifactories:
  - name: dev
    checksum_header: X-Checksum-Sha1
```

Using  debug I see :
```
executing request: PUT https://my-artifactory.com/artifactory/go-local/cli-project/0.0.1/proj_0.0.1_Linux_x86_64.tar.gz (headers: map[Authorization:[Basic ***] X-Checksum-Sha256:[1429edd2107a58d5f981850.....]])
```
I must be doing something wrong, or neither is possible?

**A bit of context and reason for a question** 
When I try to release my package on Artifactory, using Sha256 only, the following warning/error appears: `Client did not publish a checksum value` (while Sha256 sum is present and correct). Even if I didn't send X-Checksum-Sha256, Artifactory (from version 5.5) natively supports SHA-256, SHA-256 checksum is calculated when it is deployed, so I'm trying to send Sha1 instead or aside. 

Many thanks in advance

**Comment by caarlos0** [comment, marked as answer]
Seems like a bug, can you open an issue instead? Thanks!

**Reply by anitabee** [reply]
Sure will do that, thanks!
